src/features/groups/components/GroupCardList/GroupCard/GroupCardContent/GroupCardDropdownMenuTriggerButton/GroupCardDropdownMenuContent/DeleteGroupDialogTriggerButton/DeleteGroupDialogContent.tsx

```diff
@@ -39,7 +44,7 @@ export const DeleteGroupDialogContent = ({
   };
   /**
    * Handle the delete button click.
-   * If the DELETE request is successful, show a success message and close the dialog and drawer.
+   * If the DELETE request is successful, show a success message and close the dialog and drawer, and if the function executes in groupSinglePage, transitions to groupCollection page
```

Comment:
This comment is also too specific to write in the shared component.

Please either remove it or rewrite it to be more generic.

Reply:
@nick-y-ito Thank you for you re-review. I updated with more generic sentence please have a look :)